app/client/components/adminUser/adminUserList.jsx

```diff
@@ -43,76 +46,91 @@ class AdminUserList extends React.Component {
   	this.props.loadUserList()
   }
 
-  updateUserAccountPermissions(user_id, admin_tier, deactivated) {
-  	this.props.updateUser(
-  		{
-			user_id: user_id,
-			admin_tier: admin_tier,
-			deactivated: deactivated},
-			{
-
-			}
-			)
+  updateUserAccountPermissions(user_id, query, deactivated) {
+  	let userId = user_id.substring(1,); // convert 'u1' to 1
+
+  	if (query === 'resend') {
+  		this.props.updateUser({
+			invite_id: userId,
+			resend: true
+			}, {})
+		} else if (query === 'delete') {
```

Comment:
don't love the string comparison, but it's frontend exclusive, not string comparison with backend data